docs/monitoring/metrics.md

```diff
@@ -659,6 +659,8 @@ Parameters:
 - `username` - (optional) InfluxDB username used for authentication
 - `password` - (optional) InfluxDB username's password used for authentication
 - `retentionPolicy` - (optional) InfluxDB retention policy, defaults to retention policy defined on the server for the db
+- `connectTimeout` - (optional) the InfluxDB connect timeout, defaults to connect timeout is 10000 ms
+- `writeTimeout` - (optional) the InfluxDB write timeout, defaults to write timeout is 10000 ms
```

Comment:
I suggest to rephrase it, so it explicitly mentions unit of measure (milliseconds). Also, change the `defaults to connect/write timeout is ...` to `default is ...`.

For example:
```
- `connectTimeout` - (optional) the InfluxDB client connect timeout in milliseconds, default is 10000 ms
- `writeTimeout` - (optional) the InfluxDB client write timeout in milliseconds, default is 10000 ms
```